### DISCUSSION TITLE
Offering some help

### DISCUSSION THREAD

**Discussion by MasterChiefNemo** [jhy/jsoup#2116] [General]
Just wondering if I can offer some assistance to catch up on the issues and pull requests?

**Comment by codespearhead** [comment]
@jhy I'm also willing to help with that.

**Comment by MasterChiefNemo** [comment]
Any thing to assist in keeping the project secure.

**Comment by jhy** [comment]
Thanks, I appreciate everyone's help.

One thing in particular I could use help with is in helping review incoming bugs and making sure the reporter has a testcase that provides a repeatable example of their problem, and their expected result.

And also in getting a prioritized list of bugs / PRs that are real and impactful.

Happy for other suggestions as well. Ping me at jonathan@hedley.net or here.

**Reply by MasterChiefNemo** [reply]
Sorry for the late response. If you need help, where exactly can I start?

**Reply by codespearhead** [reply]
I suggest starting by [reviewing the open PRs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) and making sure they have a failing test case that passes with the patches in the PR, then we can move onto triaging the open issues.
